src/Idler/MainWindow.xaml

```diff
@@ -5,13 +5,35 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:Idler"
         xmlns:views="clr-namespace:Idler.Views"
+        xmlns:converters="clr-namespace:Idler.Converters"
         mc:Ignorable="d"
         DataContext="{Binding RelativeSource={RelativeSource Self}}"
         Title="{Binding FullAppName}"
         ResizeMode="CanResizeWithGrip">
     <Window.Resources>
         <CollectionViewSource x:Key="NoteCategoriesList"
                               Source="{Binding NoteCategories.Categories}" />
+        <converters:TotalEffortToColorConverter x:Key="TotalEffortToColorConverter" />
+        <Style x:Key="HighlightedCalendarDayButtonStyle" TargetType="{x:Type CalendarDayButton}" BasedOn="{StaticResource {x:Type CalendarDayButton}}">
+            <Style.Setters>
+                <Setter Property="Background">
```

Comment:
Number day is unreadable for `today`
<img width="127" alt="image" src="https://github.com/sergey-koryshev/idler/assets/11807074/a2411f7b-7e70-45f2-9a01-69bdeca62e18">

Reply:
Fixed